flink-contrib/docker-flink/Dockerfile

```diff
@@ -36,22 +31,24 @@ ENV PATH $PATH:$FLINK_HOME/bin
 EXPOSE 8081
 EXPOSE 6123
 
+# flink-dist can point to a directory or a tarball on the local system
+ARG flink_dist=NOT_SET
+
 # Install build dependencies and flink
+ADD $flink_dist $FLINK_INSTALL_PATH
 RUN set -x && \
-  mkdir -p $FLINK_INSTALL_PATH && \
-  apk --update add --virtual build-dependencies curl && \
-  curl -s $(curl -s https://www.apache.org/dyn/closer.cgi\?preferred\=true)flink/flink-${FLINK_VERSION}/flink-${FLINK_VERSION}-bin-hadoop${HADOOP_VERSION}-scala_${SCALA_VERSION}.tgz | \
```

Comment:
I am not talking about a published image on docker hub, but the image that exists in the flink repo, the whole reason I jumped in and worked in this image was because I was aware that the community was around the image, and it will be improved/updated, people can't be aware of the image existance/maintenance if we don't show it somewhere.

Ok for the image on dataArtisans it makes sense but I have to be honest I would prefer it to be still at Apache, at least I hope it preserves the license (I hadn't seen you work there, nice).

And yes, when you propose an image, you have to show that you plan to maintain it, or that you are part of the community.
https://github.com/docker-library/official-images#maintainership
This is important because they (docker) want images to get security updates / improvements.